Gemfile

```diff
@@ -27,6 +28,7 @@ else
   gem 'activemodel', gem_version
   gem 'actionpack', gem_version
   gem 'activerecord', gem_version, group: :test
+  gem 'rails-controller-testing' if version > '4.2'
```

Comment:
do you mean >= 4.2?

Reply:
not exactly sure and I cant seem to figure out where is this test metric at. Based on the gem file, rails-controller-testing is only added when version is set to 'master'.  

I dont think 0.9-stable has ever been configured to work with the offical release of rails 5 and above.